senderworker/src/main/scala/com/gu/notifications/ec2worker/SenderWorker.scala

```diff
@@ -2,60 +2,60 @@ package com.gu.notifications.ec2worker
 import com.amazonaws.services.lambda.runtime.events.SQSEvent
-import com.gu.notifications.worker.tokens.ChunkedTokens
-import models.BreakingNewsNotification
-import java.util.UUID
-import models.GITContent
-import models.Link
-import models.ShardRange
-import models.Topic
-import models.TopicTypes
-import models.Importance
-import play.api.libs.json.Json
-import java.time.Instant
-import models.NotificationType
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
 import scala.jdk.CollectionConverters._
 import org.slf4j.LoggerFactory
 import org.slf4j.Logger
 import _root_.models.{Android, AndroidBeta, AndroidEdition, Ios, IosEdition, Platform}
+import com.amazonaws.services.lambda.runtime.Context
+import com.amazonaws.services.sqs.{AmazonSQSAsyncClientBuilder, AmazonSQSClientBuilder}
+import com.amazonaws.services.sqs.model.{Message, ReceiveMessageRequest}
 import com.gu.notifications.worker.IOSSender
 import com.gu.notifications.worker.AndroidSender
+import com.gu.notifications.worker.utils.Aws.credentialsProvider
+
+import scala.concurrent.Future
 
 object SenderWorker extends App {
 
   implicit val logger: Logger = LoggerFactory.getLogger(this.getClass)
 
-  val notification = BreakingNewsNotification(
-    id = UUID.randomUUID(),
-    `type` = NotificationType.BreakingNews,
-    title = Some("French president Francois Hollande says killers of Normandy priest claimed to be from Islamic State"),
-    message = Some("French president Francois Hollande says killers of Normandy priest claimed to be from Islamic State"),
-    thumbnailUrl = None,
-    sender = "matt.wells@guardian.co.uk",
-    link = Link.Internal(
-      "world/2016/jul/26/men-hostages-french-church-police-normandy-saint-etienne-du-rouvray",
-      Some("https://gu.com/p/4p7xt"),
-      GITContent,None,
-    ),
-    imageUrl = None,
-    importance = Importance.Major,
-    topic = List(Topic(TopicTypes.Breaking, "uk"), Topic(TopicTypes.Breaking, "us"), Topic(TopicTypes.Breaking, "au"), Topic(TopicTypes.Breaking, "international")),
-    dryRun = Some(true)
-  )
-
-  val tokens = ChunkedTokens(
-    notification = notification,
-    range = ShardRange(0, 1),
-    tokens = List("token")
-  )
-
-  val sqsEvent: SQSEvent = {
-    val event = new SQSEvent()
-    val sqsMessage = new SQSMessage()
-    sqsMessage.setBody(Json.stringify(Json.toJson(tokens)))
-    sqsMessage.setAttributes((Map("SentTimestamp" -> s"${Instant.now.toEpochMilli}").asJava))
-    event.setRecords(List(sqsMessage).asJava)
-    event
+  val sqsClient = AmazonSQSAsyncClientBuilder.standard()
+    .withCredentials(credentialsProvider)
+    .withRegion("eu-west-1")
+    .build()
+
+  def pollQueue(queue: String, handleChunkTokens: (SQSEvent, Context) => Unit): Future[Unit] = Future {
+    logger.info(s"About to poll queue - $queue")
+    val receiveMessages = sqsClient.receiveMessage(new ReceiveMessageRequest().withQueueUrl(queue).withWaitTimeSeconds(20)).getMessages.asScala.toList
```

Comment:
The wait time _might_ be something we want to experiment with later on, there might be some tradeoff we'd be willing to make, eg make more api calls but more quickly get messages off the queue